08-compiler-features.tex

```diff
@@ -228,7 +228,7 @@ \section{Completion}
 haxe --connect 6000 myproject.hxml
 \end{lstlisting}
 
-Please note that you can use \ic{--cwd} at the first sent commandline parameter to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
+Please, note that you can use the parameter \ic{--cwd} at the first sent command line to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
```

Comment:
This comma should definitely go though. "Please note" belong together, if we have a comma there it sounds more like "Please, for the love of god, note that" to me.